jobs/apptypes_test.groovy

```diff
@@ -39,7 +39,6 @@ job(name) {
     colorizeOutput 'xterm'
     credentialsBinding {
       string("AUTH_TOKEN", "a62d7fe9-5b74-47e3-9aa5-2458ba32da52")
-      string("CLOUD_PROVIDER", defaults.e2eRunner.provider)
```

Comment:
the last pr we merged moved it to being a stringparam but for some reason on my local machine the removal of this line was never staged into the commit. So this PR removes the lines which should have been removed on the last PR =P